situp-plugins/apmtracesource/build.gradle

```diff
@@ -7,6 +7,13 @@ dependencies {
     compile project(':situp-api')
     compile project(':situp-plugins:common')
     implementation 'io.netty:netty-all:4.1.51.Final'
+    implementation 'io.opentelemetry:opentelemetry-proto:0.6.0'
+    implementation 'com.google.protobuf:protobuf-java-util:3.11.0'
     implementation "com.fasterxml.jackson.core:jackson-databind:${versionMap.jackson_databind}"
     implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${versionMap.jackson_dataformat_yaml}"
+    testImplementation "junit:junit:${versionMap.junit}"
+    testImplementation "org.junit.jupiter:junit-jupiter-api:5.5.2"
+    testImplementation 'org.assertj:assertj-core:3.6.2'
+    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.5.2"
+    testRuntime "org.junit.platform:junit-platform-runner:1.5.2"
```

Comment:
https://github.com/junit-team/junit5-samples/blob/r5.7.0/junit5-jupiter-starter-gradle/build.gradle ?

Reply:
Will take a look at this and modify.